starknet-crypto/src/ecdsa.rs

```diff
@@ -38,6 +56,18 @@ impl core::fmt::Display for Signature {
     }
 }
 
+#[cfg(feature = "signature-display")]
+impl core::fmt::Display for ExtendedSignature {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+        write!(
+            f,
+            "{}{}",
```

Comment:
Shouldn't it also display `v`?

Reply:
Yeah you're right. I overlooked this lol. Thanks